extensions/sql-database-projects/package.json

```diff
@@ -456,6 +456,7 @@
     }
```

Comment:
doesn't the sql-database-projects extension version also need to be bumped for this release?